low/low.go

```diff
@@ -717,12 +717,12 @@ func CreateLPM(name string, socket uint8, maxRules uint32, numberTbl8 uint32, tb
 }
 
 // AddLPMRule adds one rule to LPM table
-func AddLPMRule(lpm unsafe.Pointer, ip uint32, depth uint8, nextHop uint32) int {
+func AddLPMRule(lpm unsafe.Pointer, ip common.IPv4Address, depth uint8, nextHop common.IPv4Address) int {
 	return int(C.lpm_add(lpm, C.uint32_t(ip), C.uint8_t(depth), C.uint32_t(nextHop)))
 }
```

Comment:
Also inside GetPortMACAddress

Reply:
We already have constants for number of bytes in IPv4, IPv6, MAC addresses in `common`. It is natural that the types for them appear in `common` as well.